### DISCUSSION TITLE
global configuration for biome (or per $USER)

### DISCUSSION THREAD

**Discussion by vanodevium** [biomejs/biome#2526] [General]
It would be cool to be able to create a global config for the biome, so that you don't have to create your own file in each project, but simply so that it takes the config from the global file.

If this already exists, please forgive me and show me how to set it up correctly.

**Comment by vanodevium** [comment]
But if there is own config file in the project, then use it.

**Comment by nhedger** [comment]
Biome [already does that](https://biomejs.dev/guides/how-biome-works/#configuration-file-resolution).

If you put a `biome.json` in your `$HOME` folder, and you don't put one in the root of your project, autodiscovery will find and use your _global_ configuration.

**Reply by vanodevium** [reply]
Thanx! It's awesome!
